ulog/include/cbuf_reader.h

```diff
@@ -252,7 +247,7 @@ class CBufHandlerLambda : public CBufHandlerBase {
     if (hash == CBufMsg::TYPE_HASH) {
       bool ret = cis.deserialize(msg);
       if (!ret) {
-        vlog_error(VCAT_GENERAL, "Could not deserialize message of type %s", CBufMsg::TYPE_STRING);
+        // Could not deserialize message of type `CBufMsg::TYPE_STRING`
```

Comment:
since we do not have a log callback, could we use some @Logcallback or a searchable string for when we will do the fix?